examples/specs/facet_custom.vl.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should we name it `facet_custom_header`?

Reply:
Rationale: we can customize other things in facet too (e.g., spacing between facet) this example focuses on customizing header.